examples/03_model_evaluation.ipynb

```diff
@@ -130,13 +130,13 @@
     {
```

Comment:
<p>maybe add comment, that training time is not enough for a reasonable prediction?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radionets-project/radionets/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>